valqueries-automapper/src/main/java/com/valqueries/automapper/elements/UnaryOperatorElement.java

```diff
@@ -0,0 +1,29 @@
+package com.valqueries.automapper.elements;
+
+import com.valqueries.IStatement;
+import com.valqueries.automapper.SqlDialect;
+import com.valqueries.automapper.ValqueriesQueryImpl;
+import io.ran.Property;
+
+public class UnaryOperatorElement implements Element {
+	private final ValqueriesQueryImpl<?> query;
+	private final Property<?> property;
+	private final String operator;
```

Comment:
we should consider making this an enum to avoid arbitrary operators (the scope of this class is well known within the project)